src-tauri/src/main.rs

```diff
@@ -42,6 +43,12 @@ fn get_port(state: State<'_, FreePort>) -> u16 {
 fn main() {
     let free_port = find_port(12000, 13000);
 
+    find_and_kill_processes("ad4m");
+
+    find_and_kill_processes("holochain");
```

Comment:
This will kill all the holochain process includes the ones didn't start by ad4m, which can be an issue if we use ad4min and launcher together..

Reply:
Yeah, exactly. I would be fine doing this for now, as long as we are only dogfooding, if something better is hard to do.

Did I get that right that this is a problem only on macOS and has to do with Tauri's Sidecar having a bug?
So if we are only waiting for Tauri to get fixed, we could revert this when that happened.

Reply:
For some reason killing the ad4m process kills the `holohchain` process but not the `lair-keystore` process so need to kill that process manually, previously we used to use nix but that doesn't work on windows. Not sure if upgrading `lair-keystore` from version `0.0.9` to the latest version would help or not.